.azure/templates/system_test_general.yaml

```diff
@@ -54,10 +56,11 @@ jobs:
         publishJUnitResults: true
         testResultsFiles: '**/failsafe-reports/TEST-*.xml'
         goals: 'verify'
-        options: '-Dgroups=${{ parameters.groups }} -Dit.test="${{ parameters.test_case }}" -DexcludedGroups=flaky,loadbalancer,networkpolicies -Dmaven.javadoc.skip=true -B -V -Dfailsafe.rerunFailingTestsCount=2'
+        options: '-Dgroups=${{ parameters.groups }} -Dit.test="${{ parameters.test_case }}" -DexcludedGroups=flaky,loadbalancer,networkpolicies,${{ parameters.excludedGroups }} -Dmaven.javadoc.skip=true -B -V -Dfailsafe.rerunFailingTestsCount=0'
```

Comment:
`rerunFailingTestsCount` set it to back to 2

Reply:
WIll do!